docs/general/getting-started.mdx

```diff
@@ -39,14 +39,16 @@ Skip Go API includes several different endpoint types which allow developers to
 
 ## 3 Basic Ways to Use the Skip Go API
 
-There are 3 ways to leverage Skip Go API to build cross-chain swapping & transferring experiences, depending on whether you're optimizing for control or for integration speed: 
-
-1. **Most control, slowest:** Use REST endpoints to have low-level control over your interactions
-2. **High control, medium speed:** Use the [Skip Go Client](/client/getting-started) Typescript Package (`@skip-go/client`) to abstract the complexity of making HTTP calls and access related helper functionality. 
-3. **Medium control, fastest:** Embed the Skip Go Widget in your frontend in a single line of code to launch with no developer effort.
-
-<Info>
- **Have questions or feedback? Help us get better!**
- 
- Join [our Discord](https://skip.build/discord) and select the "Skip Go Developer" role to share your questions and feedback.
-</Info>
\ No newline at end of file
+There are 3 ways to leverage Skip Go API to build cross-chain swapping & transferring experiences, depending on whether you're optimizing for control or for integration speed.
+
+<CardGroup cols={3}>
+  <Card title="Rest Endpoint — Most control, slowest" icon="sliders" href="/general/overview-and-typical-usage">
+    Use REST endpoints for low-level control over your interactions.
```

Comment:
This extra organization is fire, we should make a note/linear ticket to turn this api one into a "Getting Started" page like the others have imo, wdyt?

Reply:
nit: wdyt about adding "integration" at the end of each, so would change "Most control, slowest" to "Most control, slowest integration"

Reply:
done and done!